<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>site_ajout</class>
 <widget class="QDialog" name="site_ajout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ajouter un site</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Assets.qrc">
    <normaloff>:/icon/add-circle-fill.png</normaloff>:/icon/add-circle-fill.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#f5fff9;
color:#000502;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>169</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>156</width>
          <height>29</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:24px;
font: &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>Ajouter un site</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QFrame" name="input_4">
     <property name="minimumSize">
      <size>
       <width>361</width>
       <height>79</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>79</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>341</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>341</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:16px;
font: &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nomDuSiteAjout">
        <property name="minimumSize">
         <size>
          <width>341</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>341</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:8px;
border:1px solid #333333;
font-size:16px;
font: &quot;Arial&quot;;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QFrame" name="input_5">
     <property name="minimumSize">
      <size>
       <width>361</width>
       <height>79</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>79</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>341</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>341</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:16px;
font: &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>Lieu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lieuDuSiteAjout">
        <property name="minimumSize">
         <size>
          <width>341</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>341</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:8px;
border:1px solid #333333;
font-size:16px;
font: &quot;Arial&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QFrame" name="input_6">
     <property name="minimumSize">
      <size>
       <width>361</width>
       <height>79</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>79</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>341</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>341</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:16px;
font: &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>Tarif journalier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tarifJournalieDuSiteAjout">
        <property name="minimumSize">
         <size>
          <width>341</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>341</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:8px;
border:1px solid #333333;
font-size:16px;
font: &quot;Arial&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>44</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>116</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius:8px;
font-size:16px;
font: &quot;Arial&quot;;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="site_annuler">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>62</y>
        <width>341</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>341</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>341</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color:#000502;
}
</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QPushButton" name="site_ajouter">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>13</y>
        <width>341</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>341</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>341</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:#98fbc1;;
}
:hover{
background-color:#07c556;
}</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Assets.qrc"/>
 </resources>
 <connections/>
</ui>
